### PR TITLE
feat: actcast device agent service client

### PR DESCRIPTION
**Pull request by notogawa**
## Check list

- [x] I wrote [CHANGELOG.md](./CHANGELOG.md) if the pull request adds/modifies features.

## Summary

Add actcast device agent service client with its "RS256" requests.

### CODE REVIEW

File: actfw_core/service_client.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        else:
+            s += c
+    return result
+
+
+def _read_bytes(conn: socket.socket, n: int) -> bytes:
+    result = b""
+    while len(result) < n:
+        result += conn.recv(1024)
+    return result
```
Comment (kenoss) [inline actfw_core/service_client.py:26]: Is it possible to share this implemenetation with `command_server.py`? (Or, should not?)

**Review by kenoss — COMMENTED** [review]
Basically LGTM.  Commented on some points.

File: actfw_core/service_client.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    def _sendrecv(self, command_id: int, payload: str) -> str:
+        cmd = f"{self._request_id} {command_id} {len(payload)} {payload}".encode("ascii")
+        self._request_id += 1
+        sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+        socket_path = os.environ["ACTCAST_SERVICE_SOCK"]
+        sock.connect(socket_path)
+        sock.sendall(cmd)
+        [request_id, command_id, response_length] = map(int, _read_tokens(sock, 3))
+        response = _read_bytes(sock, response_length)
+        sock.close()
```
[the remark this answers is not in the document]
Comment (notogawa) [reply actfw_core/service_client.py:33]: By an application and agent IPC spec.

File: actfw_core/service_client.py (hunk 1 above)
Comment (notogawa) [reply actfw_core/service_client.py]: yes 1b29571

**Review by kenoss — APPROVED** [review]
LGTM

**Pull request merged**